.jenkins/build_docker_images.Jenkinsfile

```diff
@@ -5,47 +5,46 @@ DCAPTOOLS_REPO = "https://dcapdockerciregistry.azurecr.io"
 DCAPTOOLS_REPO_CREDENTIAL_ID = "dcapdockerciregistry"
 
 def buildDockerImages() {
-    node("nonSGX") {
-        timeout(GLOBAL_TIMEOUT) {
-            stage("Checkout") {
-                cleanWs()
-                checkout scm
-            }
+	node("nonSGX") {
+		timeout(GLOBAL_TIMEOUT) {
+			stage("Checkout") {
+				cleanWs()
+				checkout scm
+			}
 			
-            def dcap = load pwd() + "/.jenkins/src/Dcap.groovy"
+			def dcap = load pwd() + "/.jenkins/src/Dcap.groovy"
 			
-            stage("Build Ubuntu 16.04 Docker Image") {
-                docker.withRegistry(DCAPTOOLS_REPO, DCAPTOOLS_REPO_CREDENTIAL_ID) {
-                    azDcapTools1604 = dcap.dockerImage("az-dcap-tools-16.04:${DOCKER_TAG}",
-                                                     ".jenkins/Dockerfile",
-                                                     "--build-arg UNAME=\$(id -un) --build-arg ubuntu_version=16.04")
-                }
-                pubazDcapTools1604 = dcap.dockerImage("oeciteam/az-dcap-tools-16.04:${DOCKER_TAG}",
-                                                    ".jenkins/Dockerfile",
-                                                    "--build-arg UNAME=\$(id -un) --build-arg ubuntu_version=16.04")
-            }
-            stage("Build Ubuntu 18.04 Docker Image") {
-                docker.withRegistry(DCAPTOOLS_REPO, DCAPTOOLS_REPO_CREDENTIAL_ID) {
-                    azDcapTools1804 = dcap.dockerImage("az-dcap-tools-18.04:${DOCKER_TAG}",
-                                                     ".jenkins/Dockerfile",
-                                                     "--build-arg UNAME=\$(id -un) --build-arg ubuntu_version=18.04")
-                }
-                pubazDcapTools1804 = dcap.dockerImage("oeciteam/az-dcap-tools-18.04:${DOCKER_TAG}",
-                                                    ".jenkins/Dockerfile",
-                                                    "--build-arg UNAME=\$(id -un) --build-arg ubuntu_version=18.04")
-            }
-            stage("Push to DCAP Docker Registry") {
-                docker.withRegistry(DCAPTOOLS_REPO, DCAPTOOLS_REPO_CREDENTIAL_ID) {
-                    azDcapTools1604.push()
-                    azDcapTools1804.push()
-                    if(params.TAG_LATEST == true) {
-                        azDcapTools1604.push('latest')
-                        azDcapTools1804.push('latest')
-                    }
-                }
-            }
-        }
-    }
+			parallel "Build Ubuntu 18.04 Docker Image": {
+				stage("Build Ubuntu 18.04 Docker Image") {
+					azDcapTools1804 = dcap.dockerImage("oetools-18.04:${DOCKER_TAG}",
```

Comment:
This file builds a docker image such as https://hub.docker.com/r/oeciteam/az-dcap-tools-18.04 so you would not want to change this to  `oetools` here. You would want to change this file instead https://github.com/microsoft/Azure-DCAP-Client/blob/master/.jenkins/Dockerfile#L6 to pull `oetools` instead of `oetools-full`

Reply:
Please take a look at this comment again as I do not believe it is resolved.

Your change will build .jenkins/Dockerfile and tag it `oeciteam/oetools...` instead of `az-dcap-tools`. What uses `az-dcap-tools`? If this Jenkinsfile changes I would expect an related change for anything that uses the image.